sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java

```diff
@@ -141,6 +143,34 @@ public CaseDetailedTableViewSteps(
               "Reporting user is not correct");
           softly.assertAll();
         });
+
+    When(
+        "I back to Case Directory using case list button",
+        () -> webDriverHelpers.clickOnWebElementBySelector(BACK_TO_CASES_LIST_BUTTON));
+
+    When(
+        "I check if Case date format is correct",
```

Comment:
Specify from where we take the date to check it, can be in multiple places, and which format (german) to avoid confusions with existing one or what else can come next